File: app/src/main/java/com/example/genshindomaindb/DBHelper.java
package com.example.genshindomaindb;

import android.content.Context;

import com.readystatesoftware.sqliteasset.SQLiteAssetHelper;

public class DBHelper extends SQLiteAssetHelper {
    public static final String DBNAME = "domains.db";
    public static final int DBVERSION = 1;

    public DBHelper(Context context)
    {
        super(context, DBNAME ,null, DBVERSION);

    }
}

File: app/src/main/java/com/example/genshindomaindb/DBHelper.kt
package com.example.genshindomaindb

import android.content.Context
import com.readystatesoftware.sqliteasset.SQLiteAssetHelper

class DBHelper(context: Context) : SQLiteAssetHelper(context, DB_NAME, null, DB_VERSION) {
    companion object
    {
        val DB_NAME : String = "domains.db"
        val DB_VERSION : Int = 1
    }
}